composer.json

```diff
@@ -22,7 +22,8 @@
     ],
     "require": {
         "php": "^7.1",
-        "doctrine/dbal": "^2.6"
+        "doctrine/dbal": "^2.6",
+        "doctrine/persistence": "^1.3"
```

Comment:
Please use the latest `1.3`, the previous ones are quite buggy (my fault mostly)